### PR TITLE
Initial commit for ec2 close port for 1433, 1521, 20, 21, 23, 27017, …

### PR DESCRIPTION
**Pull request by lytran2000**
…3306, 5439, 5601, 8080, 9200 and 9300.
The code were replicated from ec2 close port 22.  Please review and approve for checkins.

### CODE REVIEW

**Comment by lytran2000** [comment]
I have include the unit test for each of the script, update tox.ini, and ran the tox to ensure the unit tests passed.
I noticed there were errors from other unit tests not relating to my checkin.  See the test result that I attach
[tox_result.txt](https://github.com/vmware-samples/secure-state-remediation-jobs/files/6462612/tox_result.txt)
 here.  I would like to submit the README in a separate PR since it is insanely painful to modify it manually.

**Comment by lytran2000** [comment]
Add changes to the README.md to the PR.

File: README.md [hunk 1]
```diff
@@ -134,6 +134,17 @@ The table below lists all the supported jobs with their links.
...
 |   22.  	| 09639b9d-98e8-493b-b8a4-916775a7dea9 	|            SQS queue policy should restricted access to required users            	|            [aws-sqs-queue-publicly-accessible](remediation_worker/jobs/aws_sqs_queue_publicly_accessible)            	|
 |   23.  	| 1ec4a1f2-3e08-11eb-b378-0242ac130002 	| Network ACL should restrict administration ports (3389 and 22) from public access 	| [aws-ec2-administration-ports-ingress-allowed](remediation_worker/jobs/aws_ec2_administration_ports_ingress_allowed) 	|
+|   24.   	|       5c8c263d7a550e1fb6560c39        |            EC2 instance should restrict public access to SSH port (20)            	|                            [ec2-close-port-20](remediation_worker/jobs/ec2_close_port_20)                            	|
+|   25.   	| 4823ede0-7bed-4af0-a182-81c2ada80203  |            EC2 instance should restrict public access to SSH port (5601)            	|                            [ec2-close-port-5601](remediation_worker/jobs/ec2_close_port_5601)                            |
```
Comment (kshrutik) [inline README.md:138]: Rule Name for port 5601 is this - EC2 instance should restrict public access to Kibana port (5601) 
Correct the rule names for all the other entries as well. Also, check if you are using the correct rule name and rule id in all the readme files.

Comment (lytran2000) [reply README.md]: sorry my bad... i copied and pasted from port 22.  I updated the README accordingly to individual port's README.
Thanks for catching this.

**Review by kshrutik — APPROVED** [review]

**Review by nandeshguru — APPROVED** [review]

**Pull request merged**